index.html

```diff
@@ -0,0 +1,104 @@
+      <ul>
+        <a class="nav-link" href="#features">Features</a>
+        <a class="nav-link" href="#how-it-work">How It Works</a>
+        <a class="nav-link" href="#pricing">Pricing</a>
+      </ul>
+    </nav>
+  </header>
+
+  <div id="container">
+    <section class="message-box">
+      <h1>Bluetooth Mouse for Apple products</h1>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <input id="email" type="email" name="email" placeholder="example@mail.com"><br>
+      <input id="submit" type="submit" value="Get Started" class="btn"></input>
+      </form>
+    </section>
+
+    <section id="features">
+      <div class="feature">
+        <i class="fab fa-free-code-camp fa-3x"></i>
+        <div>
+          <h2>Overview</h2>
+          <p>Magic Mouse 2 is completely rechargeable, so you’ll eliminate the use of traditional batteries. It’s lighter, has fewer moving parts thanks to its built-in battery and continuous bottom shell, and has an optimized foot design — all helping Magic Mouse 2 track easier and move with less resistance across your desk. And the Multi-Touch surface allows you to perform simple gestures such as swiping between web pages and scrolling through documents. Magic Mouse 2 is ready to go right out of the box and pairs automatically with your Mac.</p>
+        </div>
+      </div>
+      <div class="feature">
+        <i class="fas fa-shipping-fast fa-3x"></i>
+        <div>
+          <h2>What’s in the Box</h2>
+          <p>
+            <ul>
+              <li>Magic Mouse 2</li>
+              <li>Lightning to USB Cable.</li>
+          </p>
+        </div>
+      </div>
+      <div class="feature">
+        <i class="fas fa-battery-full fa-3x"></i>
+        <div>
+          <h2>System Requirements</h2>
+          <p>Bluetooth-enabled Mac computer with OS X v10.11 or later.</p>
+        </div>
+      </div>
+    </section>
+    <section id="how-it-work">
+      <iframe id="video" src="https://www.youtube.com/watch?v=-hExjHFst0g"></iframe>
+    </section>
+    <section id="pricing">
+      <div class="price-box">
+        <h3>White Color</h3>
+        <h2>$100</h2>
+        <p>Lorem ipsum.
+          Lorem ipsum.
+          Lorem ipsum dolor.
+          Lorem ipsum.</p>
+        <a href="#" class="btn">SELECT</a>
+      </div>
+      <div class="price-box">
+        <h3>Blue Color</h3>
+        <h2>$120</h2>
+        <p>Lorem ipsum.
+          Lorem ipsum.
+          Lorem ipsum dolor.
+          Lorem ipsum.</p>
+        <a href="#" class="btn">SELECT</a>
+      </div>
+      <div class="price-box">
+        <h3>Black Color</h3>
+        <h2>$200</h2>
+        <p>Lorem ipsum.
+          Lorem ipsum.
+          Lorem ipsum dolor.
+          Lorem ipsum.</p>
+        <a href="#" class="btn">SELECT</a>
+      </div>
+    </section>
+  </div>
+  <!-- FOOTER -->

```

Comment:
Please remove unnecessary comments in your codebase. This makes code your unclean.

Reply:
- [OPTIONAL] It's good to show why a change is necessary, now, explain why is it could improve here a little bit, you could use a positive reason in place a negative one, in place to write `This makes code your unclean` say something like `This helps you to code clean and avoid confusions to other programmers or you in the future`.